RecoLocalCalo/HcalRecAlgos/test/mahifitdebugger_cfi.py

```diff
@@ -0,0 +1,10 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
the standard style for cfi files is to have the python instance to be the same name as the file itself.
So, either rename the file to mahiDebugger_cfi,
or rename the instance inside the file to mahifitdebugger.
The former is better, IMHO.